src/courses/forms.py

```diff
@@ -123,6 +123,7 @@ def __init__(self, *args, **kwargs):
         self.helper.form_tag = False
 
         self.helper.layout = Layout(
+            Div('name', css_class='col-xs-6 col-sm-4',),
```

Comment:
In Bootstrap, there are 12 columns, so if you add a new column and want it to be equal, all three should be xs-4.  I don't know what the -sm-4 is doing there, I don't think it's doing anything an you can remove from all three.  However please check responsive behaviour by making browser wider (full screen) and narrower (phone-like) to confirm doesn't break the layout.  Reference: https://getbootstrap.com/docs/3.4/css/#grid

Reply:
got it